scipy/fft/tests/test_basic.py

```diff
@@ -91,6 +91,7 @@ def test_ifft2(self, xp):
         xp_assert_close(expect * (30 * 20), fft.ifft2(x, norm="forward"))
 
     @array_api_compatible
+    # torch.fft not yet implemented by array-api-compat
```

Comment:
This is probably fine for now--we could also perhaps benefit from having the reason passed right through to `pytest` reports so that we see:

`SKIPPED [1] scipy/conftest.py:194: torch.fft not yet implemented by array-api-compat`

This patch seems to do the trick locally:

```diff
--- a/scipy/conftest.py
+++ b/scipy/conftest.py
@@ -179,11 +179,12 @@ def skip_if_array_api_gpu(func):
     return wrapped
 
 
-def skip_if_array_api_backend(backend):
-    def wrapper(func):
-        reason = (
-            f"do not run with Array API backend: {backend}"
-        )
+def skip_if_array_api_backend(backend, reason=None):
+    def wrapper(func, reason=reason):
+        if reason is None:
+            reason = (
+                f"do not run with Array API backend: {backend}"
+            )
         # method gets there as a function so we cannot use inspect.ismethod
         if '.' in func.__qualname__:
             @wraps(func)
diff --git a/scipy/fft/tests/test_basic.py b/scipy/fft/tests/test_basic.py
index b459226cc..eeac6cb9d 100644
--- a/scipy/fft/tests/test_basic.py
+++ b/scipy/fft/tests/test_basic.py
@@ -91,8 +91,8 @@ class TestFFT1D:
         xp_assert_close(expect * (30 * 20), fft.ifft2(x, norm="forward"))
 
     @array_api_compatible
-    # torch.fft not yet implemented by array-api-compat
-    @skip_if_array_api_backend('torch')
+    @skip_if_array_api_backend('torch',
+            reason="torch.fft not yet implemented by array-api-compat")
     def test_fftn(self, xp):
         x = xp.asarray(random((30, 20, 10)) + 1j*random((30, 20, 10)))
         expect = fft.fft(fft.fft(fft.fft(x, axis=2), axis=1), axis=0)
```

Reply:
Copying this comment to gh-19181.

Reply:
LGTM to move it there (and +1 for the idea) - resolving.